routing/http/server/server.go

```diff
@@ -133,12 +143,28 @@ func Handler(svc ContentRouter, opts ...Option) http.Handler {
 		opt(server)
 	}
 
+	if server.promRegistry == nil {
+		server.promRegistry = prometheus.NewRegistry()
+	}
+
+	// Create middleware with prometheus recorder
+	mdlw := middleware.New(middleware.Config{
+		Recorder: metrics.NewRecorder(metrics.Config{
+			Registry: server.promRegistry,
+			Prefix:   "delegated_routing",
```

Comment:
nit: we may have client and server metrics  in one process, let's make clear this is from server
```suggestion
			Prefix:   "delegated_routing_server",
```